src/Compilers/Core/Portable/Symbols/ISymbolExtensions_IsAccessibleWithin.cs

```diff
@@ -0,0 +1,291 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
i am very suprised github didn't see this as a move.  Did you change a lot of code in here?

Reply:
I didn't move it. I rewrote it from scratch to implement the language semantics.

Reply:
> I didn't move it. I rewrote it from scratch to implement the language semantics.

Great!  Then that removes my concern about language semantics not matching.  Can you confirm these are the right semantics for VB and for C#?  Thanks!